Screenbox/Pages/SettingsPage.xaml

```diff
@@ -113,6 +113,7 @@
                 Padding="{StaticResource PageContentMargin}">
                 <StackPanel.ChildrenTransitions>
```

Comment:
Shouldn't both transitions be inside a ```<TransitionCollection>```?

Reply:
The `<TransitionCollection>` considers the whole panel, in this case `StackPanel`, as one entity to calculate the animation. So when the cards change their position, there won't be any animation since from the `RepositionThemeAnimation` perspective, the panel only changes in size and not position. 

`EntranceThemeAnimation` would work here, but `IsStaggeringEnabled` would not have the staggered animation effect.

Reply:
No need. It's implicitly defined. See [ChildrenTransitions ](https://learn.microsoft.com/en-us/uwp/api/windows.ui.xaml.controls.panel.childrentransitions#remarks)

Reply:
Good to know, thanks for the link